src/main/java/uk/gov/hmcts/ccd/domain/service/createevent/CreateCaseEventService.java

```diff
@@ -216,7 +226,7 @@ private void saveAuditEventForCaseDetails(final AboutToSubmitCallbackResponse ab
         auditEvent.setUserId(user.getId());
         auditEvent.setUserLastName(user.getSurname());
         auditEvent.setUserFirstName(user.getForename());
-        auditEvent.setCreatedDate(LocalDateTime.now(ZoneOffset.UTC));
+        auditEvent.setCreatedDate(now());
```

Comment:
createdDate in the audit event may potentially be different from newCaseDetails.setLastStateModifiedDate(now).
Would this not be a problem?

Reply:
I thought about it. But I guess I don't think it'll give a problem. The difference is in just nano seconds. Similar thing we've already in `setLastModified` in  
https://github.com/hmcts/ccd-data-store-api/blob/master/src/main/java/uk/gov/hmcts/ccd/data/casedetails/DefaultCaseDetailsRepository.java#L70
Having said I'm happy to fix for `LastStateModified`. Please let me know.

Reply:
Done :)

Reply:
As long as there is no and will be no logic in code or in data analysis/reporting that assumes strict equality of the values in both places, nano seconds wouldn't really matter. But my worry was around potential later issues with codes or data analytics assuming strict equality. Thanks for the improvement.